daft/dataframe/dataframe.py

```diff
@@ -1546,6 +1546,17 @@ def max(self, *cols: ColumnInputType) -> "DataFrame":
 
         return self.df._agg([(c, "max") for c in cols], group_by=self.group_by)
 
+    def any_value(self, *cols: ColumnInputType) -> "DataFrame":
+        """Returns an arbitrary value on this GroupedDataFrame
```

Comment:
You should document if the values for multiple columns are guaranteed to be from the same row